Makefile

```diff
@@ -104,6 +104,8 @@ pylint:
 # Fix Python files that are not properly formatted.
 pyformat:
 	pre-commit run black --all-files
+	pre-commit run isort --all-files
+	pre-commit run no-relative-imports --all-files
```

Comment:
`no-relative-imports` is only a check, or? Not sure if we should run this on here since the other commands are actually automatically fix wrong formatting/sorting.

Reply:
Removed.